FordLabs/PeopleMover
# View-Only Modals Part II - Electric Boogaloo

WIP for #95 

## What was done
- [x] Modals for view only and edit access were split
- [x] View-only access modal was commented out to create the appearance that nothing has changed compared to production

## How to test
1. Navigate to a space
2. In the top right menu, click "Share Access"
3. Notice that the dialog has not changed significantly from previous versions. The text area for emails is now resizable and has a minimum height.
